build/index.sh

```diff
@@ -3,75 +3,84 @@ set -e
 cd $(dirname $0)
 
 set +e
-tar_exec=$(command -v gtar)
+tar_exec=$(command -v gtar) --wildcards --ignore-case
 if [ $? -ne 0 ]; then
 	tar_exec=$(command -v tar)
 fi
 set -e
 echo using tar executable at $tar_exec
 
 download () {
-	curl -L -# --compressed -A 'https://github.com/eugeneware/ffmpeg-static' -o $2 $1
+  if [ -e $2 ]; then
```

Comment:
I'd only accept a logic that does not download again, if it compares `Last-Modified` or `ETag` to make sure there isn't a new/different binary available.

Reply:
agreed, I'll probably take this code out since I was just using it to test locally without downloading each time :)